app/src/main/java/com/termux/api/apis/ClipboardAPI.java

```diff
@@ -35,7 +37,14 @@ protected boolean trimInput() {
 
                     @Override
                     public void writeResult(PrintWriter out) {
-                        clipboard.setPrimaryClip(ClipData.newPlainText("", inputString));
+                        var clipData = ClipData.newPlainText("", inputString);
```

Comment:
I am not familar with how you prefer to write Java, but I prefer to use implicit types where available in strictly typed languages. var should do the trick instead of having to write large classnames, or you want me to change this to maintain consistency of smell with other parts of the code?